static/styles/compose.css

```diff
@@ -525,3 +551,56 @@ a.compose_control_button.hide {
         margin-left: 5px;
     }
 }
+
+#compose.compose-fullscreen {
+    height: 100%;
+    z-index: 99;
+}
+
+.compose-fullscreen {
```

Comment:
Should these two headings both be `#compose.compose-fullscreen`?  You can have properties in the top-level element in an SCSS block.

Reply:
Yes, my mistake. Fixed this.